cmd/thanos/rule.go

```diff
@@ -72,6 +73,8 @@ func registerRule(m map[string]setupFunc, app *kingpin.Application, name string)
 	gcsBucket := cmd.Flag("gcs.bucket", "Google Cloud Storage bucket name for stored blocks. If empty, ruler won't store any block inside Google Cloud Storage.").
 		PlaceHolder("<bucket>").String()
 
+	externalQueryURL := cmd.Flag("external.query.url", "The external Thanos Query URL").PlaceHolder("<external-query-url>").String()
```

Comment:
I would be really specific here so maybe "alert.query-url", "The external Thanos Query URL that would be set in all alerts `Source` field.